src/test/java/edu/cornell/kfs/sys/service/impl/AwsSecretServiceImplIntegrationTest.java

```diff
@@ -78,12 +98,36 @@ void testBooleanSetAndGet() {
     }
     
     @Test
-    void testPojo() throws JsonMappingException, JsonProcessingException {
+    void testBooleanSetAndGetWithOutCache() {
```

Comment:
Since you added the extra wait to the related POJO-without-cache method, should this method also have the extra wait?

Reply:
can't hurt

Reply:
So, if the delay is needed to make the POJO tests work because they use the same secret, then should the delay be added to this secret boolean test as well? The two boolean tests also use the same secret.

Reply:
I hadn't seen issues with that test in terms of latentency issues, but it would be good to add it for consistency, thanks for noticing.